spring-xd-rxjava/src/test/resources/org/springframework/xd/rxjava/rxjava.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:int="http://www.springframework.org/schema/integration"
+       xmlns="http://www.springframework.org/schema/beans"
+       xsi:schemaLocation="http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd
+		http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+
+    <int:channel id="outputChannel1">
+        <int:queue capacity="20"/>
+    </int:channel>
+
+    <int:channel id="outputChannel2">
+        <int:queue capacity="20"/>
+    </int:channel>
+
+
+    <int:channel id="toMessageHandlerChannel"/>
+
+    <int:channel id="toStringHandlerChannel"/>
+
+    <bean id="messageProcessor" class="org.springframework.xd.rxjava.PongMessageProcessor"/>
+
+    <!--<bean name="reactorMessageHandler" class="org.springframework.xd.rxjava.SubjectMessageHandler">-->
```

Comment:
Should clean both the comment and the beans, are they are now defined in files that bear the name of the test class if I understand correctly.
Will fix on merge